src/api/atlas_sdk_client.js

```diff
@@ -750,7 +762,8 @@ class AtlasSDKClient {
         'X-Sales-Channel': this.config.salesChannel
       }, headers),
       body: json,
-      transform: (response) => new CartResponse(response)
+      transform: (response) => new CartResponse(response),
+      rawResponse: true
```

Comment:
This should be optional to the consumer of `createCheckoutCart`. The default should be false for backward compatibility

Reply:
I suggest to refactor this method to accept an `options` as the third input. The `options` default value can be something like that:
```
{
  headers: {},
  rawResponse: false
}
```